docker/docker-compose.yml

```diff
@@ -0,0 +1,94 @@
+# This is a locally modified version of the upstream CKAN docker compose file.
+
+volumes:
+  ckan_config:
+  ckan_home:
+  ckan_storage:
+  ckan_coveragedata:
+  pg_data:
+  solr_data:
+  nginx_config:
+  ckan_api_key:
```

Comment:
This volume can be removed (it is needed in EPOS-MSL for transferring the API key to MSL-API)